spec/modulr/api/transfers_service_spec.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+RSpec.describe Modulr::API::TransfersService, :unit, type: :client do
+  subject(:transfers) { described_class.new(client) }
+
+  let(:client) { instance_double(Modulr::Client) }
+  let(:response) { Struct.new(body: , keyword_init: true) }
+  let(:modulr_response) { response.new(body: { data: fixture_response }) }
+  let(:options) { {} }
+  let(:fixture_response) do
```

Comment:
Do we want a fixture system to check modulr API responses?

Reply:
The mocking system that we have in Devengo Client seems cleaner to me https://github.com/devengoapp/devengo_client/blob/81cc3384a473e9fdf88fd362b540dca59b12ca50/spec/lib/core/services/authentication_service_spec.rb#L20

Reply:
I mentioned in Slack that I want all these client gems to evolve to use `webmock` and HTTP files cf. https://devengo.slack.com/archives/CHLUNJ0KS/p1658003395168969?thread_ts=1658003275.097419&cid=CHLUNJ0KS

Reply:
I think you mean the same we maintain in devengo_client gem, where we mock all real request with webmock and return response through fixtures. If it is the case I'll do in another PR due to the changes we need to add to implement it 👍🏻